log/log.go

```diff
@@ -151,7 +151,7 @@ func (l *Logger) log(v Level, w io.Writer, msg interface{}, args ...interface{})
 	l.Lock()
 	defer l.Unlock()
 	// }
-	if v >= l.level {
+	if v >= l.level && int(v) < len(levels) {
```

Comment:
@mattn I was just wondering, what's the condition for crash?, The value of `v` is bound from 0-7 and it `levels[v]` will never crash. I have reverted the changed but if you think it is still valid please open the PR.